discovery-provider/src/queries/get_saves.py

```diff
@@ -50,6 +50,7 @@ def get_saves(save_type, user_id):
                     session.query(Track.track_id).filter(Track.is_current == True)
                 )
             )
+        query = query.order_by(Save.blocknumber.desc())
```

Comment:
Per @isaacsolo 's message, you'd want to do
```
query = query.order_by(
  Save.blocknumber.desc(),
  Save.save_item_id.desc() # or asc()
)
```

In case two saves are in the same block to make sure that the ordering is consistency

Reply:
oh, yep you're right